### PR TITLE
[SPRITE|MAP] Abductor Sprite and Animation Change

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR slightly changes the Abductor Agent's sidearm and baton visually, as well as adding in new sprites for the alien hand cuff box and piece of paper, the changes to the gun and gaton mainly include a animation of the guns charge level and the baton's current mode. Also new cuff sprites.

The main change to the gun, aside from the animations, is that the barrel is slightly elongated to accommodate two additional ammo charge lines. 

Images provides
Here is a in game image of all the charge levels
screenshot 2015-07-15 02 50 44

This is the 'empty' animation.
ayylienempty

Image of the stun batons, mob and obj.
screenshot 2015-07-15 12 03 36
Probe..........Cuff...........Sleep.......Stun

Image of the new alien paper and handcuff box.
screenshot 2015-07-15 15 08 33

Alien Hand Cuffs

### CODE REVIEW

**Comment by Steelpoint** [comment]
Pinging @AnturK as I am touching his code baby.

**Comment by AnturK** [comment]
All good :+1:
Could you also add some colored light on the baton to differentiate between modes while you're at it ?

**Comment by Steelpoint** [comment]
Of course, I'll do that tomorrow.

On that I'll likely only colour in the purple outer prod, my suggestion for the different colours would be..

* Yellow for stun
* Blue for sleep
* Red for restrain
* Purple for probe

**Comment by Steelpoint** [comment]
Updated the alien stun baton's sprites to reflect what function they are in in game, image below.

screenshot 2015-07-15 12 03 36

**Comment by Steelpoint** [comment]
New additions.

I've resprited the piece of paper and the hand cuff box into something a little more alien looking.

screenshot 2015-07-15 15 08 33

As a added bonus the alien paper is transparent, so you can somewhat see through it.

**Comment by AnturK** [comment]
:+1: Very nice.

**Comment by Steelpoint** [comment]
New handcuff sprites.

screenshot 2015-07-15 19 31 35

File: code/game/gamemodes/abduction/abduction_gear.dm [hunk 1]
```diff
@@ -301,12 +301,14 @@
 	icon_state = "alienpistol"
 	item_state = "alienpistol"
 
-/obj/item/weapon/gun/energy/decloner/alien/update_icon() // No charge levels
-	return
+//obj/item/weapon/gun/energy/decloner/alien/update_icon() // No charge levels
```
Comment (RemieRichards) [inline code/game/gamemodes/abduction/abduction_gear.dm]: remove this, don't comment it out.

**Comment by vista-** [comment]
rip pink dildo, say hi to fuzzy handcuffs

File: code/game/gamemodes/abduction/abduction_gear.dm (hunk 1 above)
Comment (Steelpoint) [inline code/game/gamemodes/abduction/abduction_gear.dm]: I was meant to, missed that sorry.

**Comment by Steelpoint** [comment]
Merge conflicts should be fixed.

**Comment by Steelpoint** [comment]
Fucking merge conflicts.

**Comment by Chocobro** [comment]
>pink probe

lewd

**Comment by Steelpoint** [comment]
Pink's the default colour guys.

**Comment by Steelpoint** [comment]
Any possibility for a merge? I can't pester on IRC due to being banned.

**Comment by duncathan** [comment]
Likely need to fix the conflicts first.

**Comment by WJohn** [comment]
Closed on request.

**Comment by AnturK** [comment]
:cry:

**Comment by Steelpoint** [comment]
I only had this closed since I was banned, now I'm unbanned I'm going to get this opened again to fix it.

**Comment by Aranclanos** [comment]
i guess

**Comment by Steelpoint** [comment]
Merge conflicts fixed.

**Pull request merged**